controller/common/node_info.py

```diff
@@ -52,5 +55,16 @@ def is_array_iscsi_iqns_match(self, host_iqns):
         host_iqns_lower = [iqn.lower() for iqn in host_iqns]
         return self._iscsi_iqn_lowercase in host_iqns_lower
 
+    def is_array_nvme_nqn_match(self, host_nqns):
+        """
+        Args:
+           host_nqns: storage host nqns list
+
+        Returns:
+           Is current host nqns matches
+        """
+        host_nqns_lower = [nqn.lower() for nqn in host_nqns]
+        return self._nvme_nqn_lowercase in host_nqns_lower
```

Comment:
could we just rename `is_array_iscsi_iqns_match`/move this to a common function?

Reply:
true. but can't we pass it as a parameter?

Reply:
sure

Reply:
@lisyukanna